cypress/integration/login.js

```diff
@@ -1,27 +1,35 @@
-describe('Sourcebooks login', function() {
-
-    it('Should display validation for empty user after attempted loggin', function () {
-        
-        cy.visit('/');
-        cy.get('.Select.not-valid').should('not.visible')
-        cy.get('[type="submit"]').click();
-        cy.get('.Select.not-valid').should('be.visible')
-    })
-
-    it('Should be able to login with role User', function () {
-
-        cy.visit('/');
-        cy.get('[id="loginForm.userId"]').click({force:true});
-        cy.get('[aria-label="Demo User"]').click();
-        cy.get('[id="loginForm.role"]').click({force:true});
-        cy.get('[aria-label="User"]').click();
-        cy.get('[type="submit"]').click();
-
-        cy.url().should('include', '/time-logging');
-        cy.get('.page__title').contains('Timesheets')
-        cy.get('.calendar').should('be.visible')
-        cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('Demo User');
-        cy.get('.main-nav').find('li').should('have.length', 1);
-    })
+import LoginPage from '../objects/loginPage';
+
+const loginPage = new LoginPage();
+
+describe('Sourcebooks login', function() {
+
+    beforeEach(function() {
+
+        cy.loginAs('User');
+        loginPage.visit();
+    })
+
+    /*it('Should display validation for empty user after attempted loggin', function () {
```

Comment:
You should not commit commented out code, if it sis commented out chances are it should be deleted